### PR TITLE
Methods for Buffer flags + Debug impl

### PR DESCRIPTION
**Pull request by astro**
Thank you for this crate!

### CODE REVIEW

File: src/buffer.rs [hunk 1]
```diff
@@ -82,6 +106,22 @@ impl Buffer{
...
+    gst_buffer_flag!(is_live, set_live, GST_BUFFER_FLAG_LIVE);
+    gst_buffer_flag!(is_decode_only, set_decode_only, GST_BUFFER_FLAG_DECODE_ONLY);
+    gst_buffer_flag!(is_discont, set_discont, GST_BUFFER_FLAG_DISCONT);
+    gst_buffer_flag!(is_resync, set_resync, GST_BUFFER_FLAG_RESYNC);
+    gst_buffer_flag!(is_corrupted, set_corrupted, GST_BUFFER_FLAG_CORRUPTED);
+    gst_buffer_flag!(is_marker, set_marker, GST_BUFFER_FLAG_MARKER);
+    gst_buffer_flag!(is_header, set_header, GST_BUFFER_FLAG_HEADER);
+    gst_buffer_flag!(is_gap, set_gap, GST_BUFFER_FLAG_GAP);
+    gst_buffer_flag!(is_droppable, set_droppable, GST_BUFFER_FLAG_DROPPABLE);
+    gst_buffer_flag!(is_delta_unit, set_delta_unit, GST_BUFFER_FLAG_DECODE_ONLY);
```
Comment (arturoc) [inline src/buffer.rs]: thanks. one thing... why is this one named different than the flag?

Comment (astro) [inline src/buffer.rs]: Well, there's a getter and a setter to each. I don't have strong preferences regarding that naming. How do you prefer it?

Comment (arturoc) [inline src/buffer.rs]: what i mean is that everything else has the same name for the get/set than the flag but in this one the flag is called `GST_BUFFER_FLAG_DECODE_ONLY` but the get/set are called `is_delta_unit`, `set_delta_unit`

Comment (astro) [inline src/buffer.rs]: Oooops! Good catch ;)

**Comment by astro** [comment]
FYI: Fixed that.

**Comment by arturoc** [comment]
thanks!

**Pull request merged**